extension/src/popup/components/identicons/AccountListIdenticon.tsx

```diff
@@ -2,94 +2,78 @@ import React from "react";
 import { useDispatch } from "react-redux";
 import styled from "styled-components";
 
-import { COLOR_PALETTE } from "popup/constants/styles";
-
 import { BasicButton } from "popup/basics/Buttons";
 
 import { truncatedPublicKey } from "helpers/stellar";
 
 import { makeAccountActive } from "popup/ducks/accountServices";
 
-import WhiteCheckIcon from "popup/assets/icon-white-check.svg";
```

Comment:
let's remove this svg from the repo as this is the only instance of it being used

Reply:
👍  https://github.com/stellar/freighter/pull/366/commits/e6a8d61e526c0b1f0e8bd971d06ffae744160a6f